R/deployApp.R

```diff
@@ -152,6 +152,7 @@ deployApp <- function(appDir = getwd(),
                       appName = NULL,
                       appTitle = NULL,
                       appId = NULL,
+                      contentId = NULL,
```

Comment:
`contentId` needs to be added to the docs above (otherwise the build will fail, I think)

Reply:
took this out.